runtime/verbose/verbose.c

```diff
@@ -586,7 +586,7 @@ printClass(J9VMThread* vmThread, J9Class* clazz, char* message, UDATA bootLoader
  */
 
 static UDATA
-parseVerboseArgument(char* options, J9VerboseSettings* verboseOptions, char** errorString) {
+parseVerboseArgument(char* options, J9VerboseSettings* verboseOptions, const char** errorString) {
```

Comment:
While you're editing things like this, please put the opening brace on a separate line.